tvaultuiv2/src/views/private/safe/components/Tree/index.js

```diff
@@ -266,6 +284,14 @@ const Tree = (props) => {
     apiService
       .deleteFolder(node.id)
       .then((res) => {
+        if (node.parentId.split('/').length === 2) {
```

Comment:
```suggestion
        if (node.parentId.split('/').length === 2) {
```

null check for node.parentId.